features/page_objects/journey/last_email_page.rb

```diff
@@ -9,6 +9,8 @@ class LastEmailPage < SitePrism::Page
   # The page will be loaded up to 10 times until the email shows
   # (a 1 in 1024 chance of the email not showing).
 
+  set_url Quke::Quke.config.custom["urls"]["last_email_bo"]
```

Comment:
Looks fine but does `order_cards.feature` still pass with this change?

Reply:
Will run that and let you know :D

Reply:
Tested, it passes :)